plugins/notifications/snoretoast/interactive.js

```diff
@@ -1,7 +1,13 @@
-const { notificationImage, icons } = require("./utils");
-const getSongControls = require('../../providers/song-controls');
-const registerCallback = require("../../providers/song-info");
-const notifier = require("node-notifier");
+const { notificationImage, icons } = require("../utils");
+const getSongControls = require('../../../providers/song-controls');
+const registerCallback = require("../../../providers/song-info");
+const WindowsToaster = require('node-notifier').WindowsToaster;
+
+const notifier = new WindowsToaster({
+    withFallback: true,
+    // see https://github.com/th-ch/youtube-music/pull/591
+    customPath: require("path").join(__dirname, 'snoretoast-x64.exe')
```

Comment:
The idea makes sense but I am not a fan of including arbitrary binaries directly in the source code of this app because they can't be verified + unless I missed something, that won't work on 32 bits, no? :)

Until the mentioned upstream PRs are merged, a workaround is to generate the custom builds in CI from a given SnoreToast version - this gives people a way to check and compile them themselves if they want - I opened https://github.com/th-ch/youtube-music/pull/600 for that - here are the generated files if you want to test with them:

[snoretoast-patched-0.8.0.zip](https://github.com/th-ch/youtube-music/files/8056235/snoretoast-patched-0.8.0.zip)